src/layout.sh

```diff
@@ -28,15 +28,18 @@ remove_listener() {
   kill_layout "$desktop"
 
   # Reset process id and layout
-  set_desktop_option $desktop 'layout' ""
-  set_desktop_option $desktop 'pid'    ""
+  set_desktop_option $desktop 'layout'
+  set_desktop_option $desktop 'pid'
 }
 
 # layout -> filename
 get_layout_file() {
   local layout_file="$LAYOUTS/$1.sh"
   # GUARD: Check if layout exists
-  [[ ! -f $layout_file ]] && echo "Layout [$layout_file] does not exist" && exit 1
+  if [ -f $layout_file ]; then
```

Comment:
actually, that is wrong :thinking: 

i tried
```bash
if [ -f $file ]; then echo "does not exist"; fi
```
and
```bash
[[ ! -f $file ]] && echo "does not exist"
```
and i do not get the same output...

the condition is simply inverted

please apply

```diff
-  if [ -f $layout_file ]; then
+  if [ ! -f $layout_file ]; then
```
:yum:

Reply:
and that is actually a major issue! :open_mouth: 

in the current state of the PR, `bsp-layout` does not work anymore :confused: 

however, applying the patch above fixes the issue and the layouts work again :wink:

Reply:
Nice catch!

Reply:
really a nice catch this totaly gone over my here 

sorry

Reply:
thanks for the bug fix :wink: